### DISCUSSION TITLE
ECHIPA COLTUL OLTENIEI

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#107] [TEAMs]
Aici sunt membrii echipei COLTUL OLTENIEI

**Comment by sebiboga** [comment]
Cristi Olteanu
role: scraper developer
discord: Cristi Olteanu

**Reply by cristiol** [reply]
NEEDED
1x Automation tester

**Comment by cristiol** [comment]
Adelina Guliman
role : Team Lead + Manual Tester
discord: adelina_85454

**Comment by cristiol** [comment]
Beatrix Racasanu
role: Tester Manual
discord: itsybixyspider

**Comment by sebiboga** [comment]
Ionut Jbanca
role: Manual Tester
discord: ionutjbk

**Comment by cristiol** [comment]
Octavian Griga
role: manual tester
discord : octavian griga

**Comment by AdrianaCionca** [comment]
Cristina Voicu
role: Manual Tester
discord: Cristina Voicu
